pkg/services/alerting/ticker.go

```diff
@@ -42,9 +42,14 @@ func NewTicker(c clock.Clock, interval time.Duration, metric *metrics.Ticker) *T
 	return t
 }
 
+func getStartTick(clk clock.Clock, interval time.Duration) time.Time {
+	nano := clk.Now().UnixNano()
+	return time.Unix(0, nano-(nano%interval.Nanoseconds()))
```

Comment:
If we are rounding the nearest 10 seconds how come we are calculating it on nanoseconds with `UnixNano` rather than `Unix` and using the first argument for `time.Unix`?